docs/source/overview.rst

```diff
@@ -1,10 +1,161 @@
-Getting started
-===============
+Getting started with Version 2.0
+================================
 
 Overview
 --------
 
-`DataTransferKit <https://github.com/ORNL-CEES/DataTransferKit>`_ is a
-library for multiphysics solution transfer.
+`DataTransferKit <https://github.com/ORNL-CEES/DataTransferKit>`_ is an
+open-source software library designed to provide parallel services for
+solution transfer for multiphysics simulations. DTK uses a general operator
+design to provide scalable parallel services for solution transfer between
+shared volumes and surfaces.
 
-Bla bla...
+DTK was originally developed at the University of Wisconsin - Madison as part
+of the Computational Nuclear Engineering Group (CNERG) group
+<http://cnerg.github.io> <https://github.com/CNERG> and is now actively
+developed at the Oak Ridge National Laboratory as part of the Computational
+Engineering and Energy Sciences (CEES) group <https://github.com/ORNL-CEES>.
+
+DTK is supported and used by the following projects and programs:
+
+* Oak Ridge National Laboratory (ORNL) Laboratory Directed Research and
+  Development (LDRD)
+  <https://www.ornl.gov/content/laboratory-directed-research-development>
+
+* Consortium for Advanced Simulation of Light Water Reactors (CASL)
+  <http://www.casl.gov>
+
+* Nuclear Energy Advanced Modeling and Simulation (NEAMS)
+  <http://www.ne.anl.gov/NEAMS/>
+
+* National Highway Traffic Safety Administration (NHTSA)
+  <http://batterysim.org>
+
+DataTransferKit Development Team
+--------------------------------
+
+DTK is developed and maintained by:
+
+* Stuart Slattery <slatterysr@ornl.gov>
+
+* Damien Lebrun-Grandie <lebrungrandt@ornl.gov>
+
+* Bruno Turcksin <turcksinbr@ornl.gov>
+
+* Andrey Prokopenko <prokopenkoav@ornl.gov>  
+
+* Roger Pawlowski <rppawlo@sandia.gov>
+
+* Alex McCaskey <mccaskeyaj@ornl.gov>
+
+
+DataTransferKit Packages
+------------------------
+
+DTK 2.0 has the following packages:
+
+**Utils**
+    General utilities for software development including exception
+    handling, MPI-based tools, and functional programming tools.
+
+**Interface**
+    Core DTK interface package. Interfaces are divided into two
+    categories: *Client* and *Operator*. *Client* interfaces define a
```

Comment:
If you meant to use inline code highlighting the correct syntax is double backticks.